src/App.js

```diff
@@ -1,36 +1,60 @@
-import React from "react";
-import TodoList from "./components/TodoComponents/TodoList.js";
-import TodoForm from "./components/TodoComponents/TodoForm.js";
+import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
 import './components/TodoComponents/Todo.css';
 
 class App extends React.Component {
-  constructor() {
-    super();
-    this.state = {
-      todoData: []
-    };
-  }
-
-  addTask = task => {
-    const newTask = {
-      task: task,
-      id: Date.now(),
-      isCompleted: false
-    };
-
-    this.setState({
-      todoData: [...this.state.todoData, newTask]
-    });
-
-    console.log(this.state.todoData);
-  };
-
+    constructor() {
+      super();
+      this.state = {
+        tasks: [],
+        holder: "",
+      }
+    }
+    toggleClass = (event, id) => {
+      event.target.classList.toggle('completed');
+      this.setState({
+        tasks: this.state.tasks.map(task => {
+          if (task.id === id) {
+            return {
+              task: task.task,
```

Comment:
Like I mentioned above you could stick with the pattern you're using and condense the code some with something like below:
``` javascript
if (task.id === id) {
          task.completed = !task.completed;
          return task;
        } else {
          return task;
        }
```
In the code above we're not overwriting the task and returning but just updating the relevant task property and then returning the task.